pcdet/models/detectors/pv_rcnn_ssl.py

```diff
@@ -342,15 +340,17 @@ def _get_proto_contrastive_loss(self, batch_dict, bank, ulb_inds):
 
 
     def _align_instance_pairs(self, batch_dict,batch_dict_pair,indices):
-        shared_ft_sa = batch_dict['shared_features_gt'].view(batch_dict['batch_size'],-1,256)[indices]
-        shared_ft_wa = batch_dict_pair['shared_features_gt'].view(batch_dict['batch_size'],-1,256)[indices]
+        
+        embed_size = 256 if not self.model_cfg['ROI_HEAD']['PROJECTOR'] else 256 # if possible, 128
```

Comment:
if and else produce the same thing! @adv010

Reply:
Yes, that could be ignored for now :) 
Currently, the setting I use is -  Both  shared_fc layer and projected_fc layer  having : 2 hidden layers with output neurons 256,256
In case we may want to try later for a lower-dimensional embedding with projected_fc layer as 256,128, then this embed_size would change to 128.